equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java

```diff
@@ -84,6 +91,18 @@ void preTest() {
                 throw new RuntimeException(e);
             }
         });
+
+        pseTLmessages = new ArrayList<>();
+        Crm14PSEMessagesModel crm14PSEMessagesModel = getCrm14PSEMessagesModel();
+        pseTLmessages.add(crm14PSEMessagesModel);
+    }
+    private static Crm14PSEMessagesModel getCrm14PSEMessagesModel() {
```

Comment:
Rename this function. It is misleading. It is better to use something like prepareMockedDataForPSE or something like that; Using a name too similar to what it would be on the service looses track of being mocked

Reply:
And all this can simply go separated into a PSE service test. And leave the mocked part to actually be mocked in here

Reply:
Removed Controller tests and added service tests